packages/components/src/ui/context/test/context-connect.tsx

```diff
@@ -10,6 +10,7 @@ import { contextConnect, contextConnectWithoutRef } from '../context-connect';
 import type { WordPressComponentProps } from '../wordpress-component';
 
 // Static TypeScript tests
+/* eslint-disable jest/expect-expect */
```

Comment:
This contains static typescript tests, so intends to use TS for checking for failures, and not Jest. While I have mixed feelings about this manifestation of tests, this intentionally doesn't contain assertion functions, so we're intentionally disabling the rule here.